test/index.js

```diff
@@ -311,10 +312,10 @@ test('S3Lambda.context.output.map (sync)', (t) => {
     bucket: bucket,
     prefix: prefix
   }
-
+  const files = ['file1big-momma', 'file2big-momma', 'file3big-momma', 'file4big-momma']
   lambda
     .context(context)
-    .output(bucket, outputPrefix, (key) => {return key.replace('-', '/')})
+    .output(bucket, outputPrefix, (key) => {return key.concat('big-momma')})
```

Comment:
with es6 you actually reduce this even more to 
```
(key) => key.concat('big-momma')
```
which is an implicit return 😉